pkg/controller/replication_controller.go

```diff
@@ -209,6 +211,14 @@ func (rm *ReplicationManager) getPodControllers(pod *api.Pod) *api.ReplicationCo
 		glog.V(4).Infof("No controllers found for pod %v, replication manager will avoid syncing", pod.Name)
 		return nil
 	}
+	if len(controllers) > 1 {
```

Comment:
It's user error to have simultaneous overlapping RCs. Eventually we'll want RC manager to put offending RCs into a "disabled" state, with a "overlaps with RC foo" status message.

For now, just always using the oldest one seems like a good strategy.